### ISSUE TITLE
Other flow teacher network?

### ISSUE THREAD

**Issue by DachunKai** [ltkong218/IFRNet#28]
May I ask if you have tried using RAFT or other methods as the teacher network? Would the results be better? How did you pretrain LiteFlowNet on Vimeo90k, and could you provide the code for this pretraining?

**Comment by ltkong218** [comment]
If replacing the teacher flow network as RAFT, the frame interpolation accuracy of IFRNet will drop. For the reason, please refer to the Task-Oriented Flow Distillation Loss in our paper. The LiteFlowNet model used in IFRNet is from [https://github.com/sniklaus/pytorch-liteflownet](https://github.com/sniklaus/pytorch-liteflownet). Thanks.

**Comment by DachunKai** [comment]
Thank you, but I couldn't find the analysis in Task-Oriented Flow Distillation Loss explaining why RAFT's teacher network is not as good as LiteFlowNet, or Pwc-Net? Additionally, I noticed that the teacher network in the RIFE paper was changed from an earlier version of LiteFlowNet to DVF. What are your thoughts on this approach?

**Comment by ltkong218** [comment]
You can refer to the paper Video Enhancement with Task-Oriented Flow (IJCV 2019). DVF is trained on the same Vimeo90K dataset while LiteFlowNet is not. Therefore using DVF's optical flow pseudo label is more helpful. It is more like an engineering trick rather than academic research.
